### PR TITLE
Fix tree builder without a root node deprecation in Symfony/Config 4.2+

### PR DESCRIPTION
**Pull request by BeyerJC**
Fixes issue #1

### CODE REVIEW

**Review by bocharsky-bw — COMMENTED** [review]
Hey @WhiteRabbitDE ,

Thank you for this PR! But you know what? Now I see we don't use `$rootNode` at all :) So, I think we should whether remove it and keep only something like this:
```php
         $treeBuilder = new TreeBuilder('bw_active_menu_item');

         return $treeBuilder;
```

Or keep your legacy fix but let's comment it out then, since we don't use it anyway. Wdyt?

**Comment by BeyerJC** [comment]
Hi @bocharsky-bw,
I'm totally with you! I saw that there is no configuration processed but was afraid to just remove it on my own without talking to you first :-)
Imho we could remove the whole part since its very unlike that it will be used someday.

Tldr : I like your solution and could update the PR If you agree.

**Comment by bocharsky-bw** [comment]
Yeah, much better, thank you! 👍
